iothub/service/src/ImportMode.cs

```diff
@@ -16,30 +16,69 @@ namespace Microsoft.Azure.Devices
     [JsonConverter(typeof(StringEnumConverter))]
     public enum ImportMode
     {
+        /// <summary>
+        /// If a device does not exist with the specified ID, it is newly registered.
+        /// If the device already exists, existing information is overwritten with the provided input data without regard to the ETag value.
+        /// </summary>
         [EnumMember(Value = "createOrUpdate")]
         CreateOrUpdate = 0,
 
+        /// <summary>
+        /// If a device does not exist with the specified ID, it is newly registered.
+        /// If the device already exists, an error is written to the log file.
+        /// </summary>
         [EnumMember(Value = "create")]
         Create = 1,
 
+        /// <summary>
+        /// If a device already exists with the specified ID, existing information is overwritten with the provided input data without regard to the ETag value.
+        /// If the device does not exist, an error is written to the log file.
+        /// </summary>
         [EnumMember(Value = "update")]
         Update = 2,
 
+        /// <summary>
+        /// If a device already exists with the specified ID, existing information is overwritten with the provided input data only if there is an ETag match.
+        /// If the device does not exist, an error is written to the log file.
```

Comment:
maybe reword "If the device does not exist or there is an ETag mismatch, an error is written to the log file.

Reply:
sure